kratos/containers/distributed_csr_matrix.h

```diff
@@ -963,7 +963,9 @@ class DistributedCsrMatrix
 
     void load(Serializer& rSerializer)
     {
-        rSerializer.load("CSRcommunicator",mrComm);
+        std::string comm_name;
+        rSerializer.load("CommunicatorName",comm_name);
+        mrComm = ParallelEnvironment::GetDataCommunicator(comm_name);
```

Comment:
hm I am not sure if this will work
maybe you need to use a pointer after all

Reply:
yes, done.
 strange thing is that it did compile locally (i don't understand how though...)